SiReCore/contexts.py

```diff
@@ -286,23 +286,35 @@ def add_basic(context_skeleton, phoneme):
   v_dct = phoneme_features.get_phoneme_feats_dict(syll.vowel_id)
   for feat in phoneme_features.get_feature_lists():
     c.add("sv"+feat, v_dct[feat])
-  
+
   ##### Word level features #####
   #print "Adding word level features to {0} in {1}".format(phoneme.id, phoneme.parent_utt.id)
   w_pos_in_utt = word.pos_in_utt()
   #Word number of phonemes
   c.add("wnp", str(word.num_phonemes()))
   #Word number of syllables
   c.add("wns", str(word.num_syllables()))
-  
+  #Word emphasis
```

Comment:
Could you add these to an extension of the basic context set instead of including them?
This is so that SiRe does not automatically try and add emphasis features by default.
You can see some of the other sets for how to do that.